test/parallel/test-buffer-indexof.js

```diff
@@ -73,3 +73,7 @@ assert.throws(function() {
 assert.throws(function() {
   b.indexOf([]);
 });
+
+// GH-1485
```

Comment:
I'd prefer a brief description of why the test. don't like needing to look up issues to see why tests were added.